package.json

```diff
@@ -60,6 +60,8 @@
     "react-dom": "^17.0.2",
     "react-ga": "^3.1.2",
     "react-gtm-module": "^2.0.11",
+    "react-helmet": "^6.1.0",
```

Comment:
We should remove it, we have for that `next/head`.

Reply:
Yes, I will remove it.